examples/access-optional-json-elements/access_optional_json_elements.out

```diff
@@ -1,2 +1,2 @@
 $ bal run working_directly_with_json.bal
```

Comment:
the file name should be changed to `access_optional_json_elements.bal` right

Reply:
Fixed.